testsuite/tests/typing-layouts-products/product_arrays.ml

```diff
@@ -124,9 +124,12 @@ let f_bad (x : #(string * float#)) = make_vect 42 x
 Line 1, characters 37-51:
 1 | let f_bad (x : #(string * float#)) = make_vect 42 x
                                          ^^^^^^^^^^^^^^
-Error: Unboxed product array elements must be external or contain all gc
-       scannable types. The product type this function is applied at is
-       not external but contains an element of sort float64.
+Error: An unboxed product array element must be formed from all
+       gc-ignorable types or all gc-scannable types. But this array
```

Comment:
```suggestion
       external types (which are ignored by the gc) or all gc-scannable types. But this array
```

@ccasin might like this use of "external" more. But I'm fine with this as-is, also.